PhysicsTools/Heppy/python/analyzers/objects/TauAnalyzer.py

```diff
@@ -81,14 +81,16 @@ def id6(tau,X):
                     loose, medium, tight, very tight, very very tight)"""
                 return id5(tau, X) + tau.tauID(X%"VVTight")
 
-            tau.idMVA = id6(tau, "by%sIsolationMVArun2v1DBoldDMwLT")
-            tau.idMVANewDM = id6(tau, "by%sIsolationMVArun2v1DBnewDMwLT")
+            tau.idMVArun2 = id6(tau, "by%sIsolationMVArun2v1DBoldDMwLT")
```

Comment:
Overall on the naming: 
At the moment, the MVA is e.g. called "MVArun2v1DBoldDMwLT". The two things that vary with the current discriminators are "with and without lifetime variables", and "oldDM vs newDM". With an updated training, I'd expect the version to increase from v1 to v2. Rather than adding "run2" to the names, we may want to let the names reflect what may actually change (the previous idea was to assume that "wLT" and "oldDM" are the defaults, and deviations from this would be reflected in the name). I'll leave it to you to decide - if you think having "run2" in the name improves the clarity, I'm also fine with it.